### PR TITLE
sign in and sign up

### PR DESCRIPTION
**Pull request by aakhare**
created ui for sign in and sign up. merged locally with main to make sure no one else's code breaks with my changes.

### CODE REVIEW

**Review by loshprung — APPROVED** [review]
Looks good. In the future, I would appreciate if some kind of README could be written with setup instructions (if that's applicable). Something like what we have under the backend directory. Would be curious to test the frontend on my own machine.

**Comment by loshprung** [comment]
@aakhare For future pull requests, please wait for at least 2 approvals before merging

**Comment by aakhare** [comment]
My bad, I clicked on it too soon -- will be more careful next time! 
Will add a README before the next request, thanks for letting me know.

**Pull request merged**
